report_qweb_signer/data/defaults.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.config_parameter" id="report_qweb_signer_java_param">
+        <field name="key">report_qweb_signer.java_parameters</field>
+        <field name="value">-Xms4M -Xmx4M</field>
```

Comment:
Try removing this default.

See https://github.com/OCA/reporting-engine/pull/253/commits/c82a549b72edcae277171f2db3518ad4183dfcd8